bin/gwd/gwd.ml

```diff
@@ -1052,7 +1052,19 @@ let authorization from_addr request base_env passwd access_type utm base_file
         basic_authorization from_addr request base_env passwd access_type utm
           base_file command
 
+let explode s =
```

Comment:
maybe `string_to_char_list` is better name

Reply:
ok